server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -137,6 +138,31 @@ public void testCopyFromFileWithUmlautsWhitespacesAndGlobs() throws Exception {
         Files.write(file.toPath(), lines, StandardCharsets.UTF_8);
 
         execute("copy t from ?", new Object[]{Paths.get(tmpFolder.toURI()).toUri().toString() + "s*.json"});
+    }
+
+    @Test
+    public void testCopyFromFileWithUmlautsWhitespacesAndGlobsWithoutScheme() throws Exception {
```

Comment:
this passes on linux but fails on windows because it tries to create `Path` from uri string containing `*`